packages/turbo-workspaces/src/managers/pnpm.ts

```diff
@@ -205,25 +205,33 @@ async function clean(args: CleanArgs): Promise<void> {
  * If this is not possible, the non pnpm lockfile is removed
  */
 async function convertLock(args: ConvertArgs): Promise<void> {
-  const { project, logger, options } = args;
+  const { project, logger, options, to } = args;
 
   if (project.packageManager !== "pnpm") {
-    logger.subStep(
-      `converting ${path.relative(
-        project.paths.root,
-        project.paths.lockfile
-      )} to pnpm-lock.yaml`
-    );
-    if (!options?.dry && existsSync(project.paths.lockfile)) {
-      try {
-        await execa("pnpm", ["import"], {
-          stdio: "ignore",
-          cwd: project.paths.root,
-        });
-      } finally {
-        rmSync(project.paths.lockfile, { force: true });
+    // pnpm does not support importing a bun lockfile
```

Comment:
Yes it does! By way of `yarn`! `bun bun.lockb` will output a `yarn.lock` file. You can execute that and then feed that into the transform.

Reply:
This also means that we can do `bun` => `yarn`.

Reply:
Done